packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -330,8 +330,9 @@ export default class NewVm extends BaseComponent {
   _reset = () => {
     this._replaceState({
       bootAfterCreate: true,
-      CPUs: '',
+      coresPerSocket: '',
       cpuCap: '',
+      CPUs: '',
```

Comment:
[`vCpus={CPUs}`](https://github.com/vatesfr/xen-orchestra/pull/4187/files#diff-a255ba66f114f6e11cd0862a0543b7dfR1093) but [`vCpus` is number](https://github.com/vatesfr/xen-orchestra/pull/4187/files#diff-e1ddd042dc76cde4a34706f4b6afd508R16).

Reply:
I updated the implementation. I'll not use `vCpu`.